BREAKING.md

```diff
@@ -36,9 +36,14 @@ This field has been deprecated and will be removed in a future breaking change.
         fileName: string,
 -       createShareLinkType?: ShareLinkTypes,
 +       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
```

Comment:
This is the second time today that I see this change in BREAKING.md. Is your repo out of sync?